golang.md

```diff
@@ -0,0 +1,119 @@
+# Golang at Clever
+
+## Golang versions
+
+All golang services (with some exceptions) at Clever should be on go-1.5.
+
+
+## Docker Images
+
+#### drone
+
+`clever/drone-go:1.5`
+
+#### runtime
+
+  - `gliderlabs/alpine:3.2` - see [catapult/Makefile](https://github.com/Clever/catapult/blob/master/Makefile) as an example. (extra build configuration needed)
+  - `google/debian:wheezy`
+
+
+## Dependencies
```

Comment:
@burnsed the reason for adding other people: it seems like this doc just talks about using Glide, through most current repos use Godeps. I'm curious if this is a decision. If not then we should add some basic usage information (or link to the doc by @cgclever) along with the glide info